src/main/java/se/sundsvall/disturbance/integration/db/converter/StatusConverter.java

```diff
@@ -0,0 +1,36 @@
+package se.sundsvall.disturbance.integration.db.converter;
+
+import static org.zalando.problem.Status.INTERNAL_SERVER_ERROR;
+
+import org.zalando.problem.Problem;
+
+import se.sundsvall.disturbance.api.model.Status;
+
+import jakarta.persistence.AttributeConverter;
+import jakarta.persistence.Converter;
+
+@Converter(autoApply = true)
+public class StatusConverter implements AttributeConverter<Status, String> {
+
+	@Override
+	public String convertToDatabaseColumn(Status attribute) {
+		if(attribute == null) {
+			return null;
+		} else {
+			return attribute.toString();
+		}
+	}
+
+	@Override
+	public Status convertToEntityAttribute(String dbData) {
+		try {
+			return Status.valueOf(dbData);
+		} catch (IllegalArgumentException e) {
+			throw Problem.builder()
```

Comment:
Is this necessary? Isn't this already in place (zalando AdviceTrait)?

Reply:
Maybe, I'll test it.